docs/rate_limited_operations.rst

```diff
@@ -62,4 +62,23 @@ You can use `rate-limit` when counting items in your table:
     # Using only 15 RCU per second
     count = User.count(rate_limit = 15):
     print("Count : {}".format(count))
+    
+Unsupported Scenarios: Rate limiting Writes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Rate limiting writes are currently not supported. One workaround for this is to keep a count that gets increased based on the time passed, and only batch that number of writes (called Leaky bucket algorithm). The **pseudocode** below assumes writes are happening constantly. In the case that they're not, sleeping during low utilization could also be useful.
+
+.. code-block:: python
+    # Note: This is pseudocode
+    def write(items):
+        while True:
+            allowance= min(
+                       allowance + time_passed/RATE_OF_WRITING, 
+                       MAX_ALLOWED_RATE
+                       )
+             number_of_items_to_be_written = min(int(allowance), len(items))
+             batch_write(items[:number_of_items_to_be_written])
```

Comment:
This doesn't take into account the size of the items being written, so I'm a little worried that someone might try to use this algorithm unsuccessfully. I wonder if it's better to just state that this is currently unsupported and add an feature request for it in issues.

Reply:
That's fair too. I suppose the workaround advice can be given offline and can be something not actually encouraged so as not to generate additional load if people try it in various configurations and get unexpected results. Updating!